### ISSUE TITLE
Some CSV files not working

### ISSUE THREAD

**Issue by wincowgerDEV** [wincowgerDEV/OpenSpecy#76]
Recently recieved this csv file that seems to be correctly formatted but doesn't work. 

Throws the error below in the logs. 
2021-05-11T16:44:17.347091+00:00 shinyapps[2419654]: Warning: Error in abs: non-numeric argument to mathematical function
2021-05-11T16:44:17.353148+00:00 shinyapps[2419654]:   142: %>%
2021-05-11T16:44:17.353148+00:00 shinyapps[2419654]:   141: adj_neg
2021-05-11T16:44:17.353149+00:00 shinyapps[2419654]:   137: adj_intens.data.frame
2021-05-11T16:44:17.353150+00:00 shinyapps[2419654]:   119: data
2021-05-11T16:44:17.353151+00:00 shinyapps[2419654]:   112: renderPlotly [/srv/connect/apps/OpenSpecy/server.R#230]
2021-05-11T16:44:17.353151+00:00 shinyapps[2419654]:   111: func
2021-05-11T16:44:17.353149+00:00 shinyapps[2419654]:   140: adj_intens.default
2021-05-11T16:44:17.353150+00:00 shinyapps[2419654]:   135: <reactive:data> [/srv/connect/apps/OpenSpecy/server.R#196]
2021-05-11T16:44:17.353152+00:00 shinyapps[2419654]:   108: shinyRenderWidget
2021-05-11T16:44:17.353177+00:00 shinyapps[2419654]:    94: renderFunc
2021-05-11T16:44:17.353177+00:00 shinyapps[2419654]:    93: output$MyPlot
2021-05-11T16:44:17.353178+00:00 shinyapps[2419654]:    13: runApp
2021-05-11T16:44:17.353178+00:00 shinyapps[2419654]:    12: fn
2021-05-11T16:44:17.353175+00:00 shinyapps[2419654]:   107: func
2021-05-11T16:44:17.353179+00:00 shinyapps[2419654]:     5: eval
2021-05-11T16:44:17.353178+00:00 shinyapps[2419654]:     7: connect$retry
2021-05-11T16:44:17.353179+00:00 shinyapps[2419654]:     6: eval

<html xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:///C:/Users/THEGRA~1/AppData/Local/Temp/msohtmlclip1/01/clip.htm">
<link rel=File-List
href="file:///C:/Users/THEGRA~1/AppData/Local/Temp/msohtmlclip1/01/clip_filelist.xml">
<style>

</style>
</head>

<body link="#0563C1" vlink="#954F72">

Wavelength;Absorbance
--
3.997.896.328;0.999979
3.995.967.772;1
3.994.039.216;0.999974
399.211.066;0.999901
3.990.182.104;0.999803
3.988.253.548;0.999699
3.986.324.993;0.999627
3.984.396.437;0.999584
3.982.467.881;0.999506
3.980.539.325;0.999387

</body>

</html>

**Comment by wincowgerDEV** [comment]
Upon further inspection, the csv file isn't formatted as a typical csv, the wavelength column has periods in it which would make those values not be read as a number. I don't know how we would support this type of data because it seems really specialized so I will just close the issue now but keep it logged in case we decide we want to reopen it later.

**Comment by zsteinmetz** [comment]
Yah, you're right. I guess we shouldn't support non-standard number formatting. Out of the box, R doesn't do so either; only after carefully adjusting the arguments of `read.csv` and alike.

**Comment by wincowgerDEV** [comment]
Yeah there are too many variants. I think we support quite a few already.
If there was an easy way to support some more I would be down but that one
seems pretty complicated.
